tasks/cleanup.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: remove EOLed opsys
+  command: faf releasemod -o "{{item|selectattr("opsys")}}" --opsys-release "{{item|selectattr("release")}}" -s EOL
+  with_items: '{{eol_opsys:|default([], true)|list}}'
+  become: yes
+  become_user: faf
+
+- name: remove EOLed packages
+  command: faf cleanup-packages "{{item|selectattr("opsys")}}" "{{item|selectattr("release")}}"
+  with_items: '{{eol_opsys:|default([], true)|list}}'
+  become: yes
+  become_user: faf
+
+# Skip whole block if faf owns atleast 1 table in db
+- block:
+  - name: create faf's database schema
+    command: faf-migrate-db --create-all
+    become: yes
+    become_user: faf
+
+  - name: stamp database as migrated to latest version
+    command: faf-migrate-db --stamp-only
+    become: yes
+    become_user: faf
+
+  - name: init faf
+    command: faf init
+    become: yes
+    become_user: faf
+  when: ( count_tables.stdout_lines[2]|int ) == 0
```

Comment:
This condition can potentially fail when you run the playbook without the `first_time_setup.yml` task, because the `count_tables` variable is defined there.

I originally thought this would be a bigger issue because before commit df236ef8e3210aef67f0990a35dc0e35e3e7c272 it was needed to set ` faf_first_time_setup: true` to run the `first_time_setup` task, but right now the task should run every time.